paper-server/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java

```diff
@@ -356,6 +356,13 @@ private void postProcess() {
         public IntOr.Default compressionLevel = IntOr.Default.USE_DEFAULT;
         @Comment("Defines the leniency distance added on the server to the interaction range of a player when validating interact packets.")
         public DoubleOr.Default clientInteractionLeniencyDistance = DoubleOr.Default.USE_DEFAULT;
+        @Comment(
+            "Defines how many orbs in an area will actually merge." +
+            "This is a percentage. Where 40 is 2.5% of the orbs will merge and 1 is 100% of the orbs will merge."
+        )
+        @Constraints.Min(1)
+        @Constraints.Max(40)
+        public int xpMergePercentage = 40;
```

Comment:
Is it possible for us to do this interpolating ourselves? As in general this 1-40 seems odd for it being called a percentage.

Reply:
Yeah I could probably do some interpolating and just floor it to the nearest int.

Reply:
It'll now interpolate the needed modulo based on a 1-100 percentage. Anything 2.5 is clamped to 40 (mostly due to it most likely doing weird shenanigans).